apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/config/PortalConfig.java

```diff
@@ -193,6 +196,16 @@ public String portalAddress() {
     return getValue("apollo.portal.address");
   }
 
+  public int refreshAdminServerAddressTaskNormalIntervalInMilli() {
+    int interval = getIntProperty("refresh.admin.server.address.task.normal.interval.ms", DEFAULT_REFRESH_ADMIN_SERVER_ADDRESS_TASK_NORMAL_INTERVAL_IN_MILLI);
```

Comment:
It's better to let the user configure the interval in seconds instead of milliseconds.